.github/workflows/tests.yml

```diff
@@ -84,3 +84,33 @@ jobs:
         run: |
           pip install typing-extensions dataclasses
           python setup.py test
+
+  python-nb-examples:
```

Comment:
very nice! I'm always excited about this kind of notebook testing.

Reply:
I can't tell from the code - what is the scope of this? Will it run all notebooks or just some?

Reply:
Ah nvm

jupyter nbconvert --to script examples/api/*.ipynb

Only notebooks in the api folder.

Reply:
Yes, although since pytest is failing on these, it is currently only running the first one:
```
pip install pytest pytest-cov nbconvert
pip install -r requirements.txt
pip install -e .
jupyter nbconvert --to script examples/api/*.ipynb
> python examples/api/start.py
```

If someone knows why pytest is failing on these tests with this command:
```
$ pytest examples/api
```

But works fine on the normal tests:
```
$ pytest
```

I would love to know. Then we can turn it back to using pytest and anything inside examples/api will be converted to .py and executed as a test.